dist/templates/ovnkube-node.yaml.j2

```diff
@@ -165,6 +165,7 @@ spec:
           runAsUser: 0
           capabilities:
             add: ["NET_ADMIN", "SYS_ADMIN", "SYS_PTRACE"]
+          privileged: true
```

Comment:
I don't think we want to make changes to the defaults here; but you could add an option to daemonset.sh to template in the changes you need only for the kind build

Reply:
Yeah so the deal is the netns is getting created in /var/run/netns in the KIND docker container. The ovn-kube pod (created as another container inside of the docker container) has not access to this path. I'm not sure how this works in a non-KIND deployment (although I know it works because I did a kubeadm deployment a while ago). I don't see how ovn-kube has access to the netns path. I see in downstream template we are mounting /var/run/netns, so I'm kind of confused here...

To fix the problem I added it as a mount volume, but it requires mountPropagation: Bidirectional, otherwise when golang netns tries to open the ns it complains that it is some file system magic going on. mountPropagation: Bidirectional requires privileged container.

I can certainly make it rendered via j2, and in fact that is the road I started down. However, after looking in the j2 files...they don't actually j2 render anything...it's just a bunch of sed statements inside daemonset.sh. That all should be fixed and the templates should actually be j2 rendered and the sed stuff abandoned. But I figured that should be a separate change. Could I do that in a follow up patch/PR?

Reply:
you don't need to fix it to be rendered via j2, but we shouldn't merge it with ovnkube-node unconditionally made privileged... Rather than making it a proper conditional you can just have it be a template field that expands to either `privileged: true` or nothing

Reply:
OK I will try to conditionalize it using the existing script.